src/reducers/FilterReducer.jsx

```diff
@@ -16,13 +16,53 @@ const filterReducerFunc = (state, action,e) => {
           return{...state,categories:{...state.categories,horror:!state.categories.horror}}; 
```

Comment:
Instead of using just "state" use something which is relevant to the state you are managing like "cartState" or something like that.